bitcoin/src/crypto/key.rs

```diff
@@ -12,7 +12,8 @@ use core::str::FromStr;
 use hashes::hex::FromHex;
 use hashes::{hash160, hex, Hash};
 use internals::write_err;
-pub use secp256k1::{self, constants, KeyPair, Parity, Secp256k1, Verification, XOnlyPublicKey};
+use rand::{Rng, thread_rng};
```

Comment:
I think the compiler would fail if this is not guarded with cfg flag too. You should be able to run with `cargo test --no-default-features`

Reply:
Done!